### PR TITLE
test: Less verbose messages about chromium in test logs

### PR DESCRIPTION
**Pull request by stefwalter**
Fix several cases where we have repetitive test logs.

### CODE REVIEW

File: test/common/testvm.py [hunk 1]
```diff
@@ -401,6 +401,7 @@ def execute(self, command=None, script=None, input=None, environment={},
             "-p", str(self.ssh_port),
             "-o", "StrictHostKeyChecking=no",
             "-o", "UserKnownHostsFile=/dev/null",
+            "-o", "LogLevel=ERROR",
```
Comment (stefwalter) [inline test/common/testvm.py]: This part looks good to me. Reviewed your changes.

**Comment by martinpitt** [comment]
I rebased the PR to fix the conflicts after #8314. While I was at it, I also removed the "Permanently added [...] to the list of known hosts" warning that cluttered every test output. Now it looks clean except for the DevTools message, which seems impossible  to get rid of:
```
❱❱❱ test/verify/check-menu 
1..1
# ----------------------------------------------------------------------
# testBasic (__main__.TestMenu)
#

DevTools listening on ws://127.0.0.1:9618/devtools/browser/a9a0b8dd-1285-4ead-8f82-74097aea68f2
ok 1 testBasic (__main__.TestMenu) # duration: 17s

# TESTS PASSED [18s on donald]
```

File: test/common/testlib.py [hunk 2]
```diff
@@ -93,7 +93,7 @@ def __init__(self, address, label, port=None, headless=True):
             self.port = port
         self.default_user = "admin"
         self.label = label
-        self.cdp = cdp.CDP("C.utf8", headless, trace=opts.trace)
+        self.cdp = cdp.CDP("C.utf8", headless, verbose=opts.trace, trace=opts.trace)
```
Comment (stefwalter) [inline test/common/testlib.py]: Oops, this should say opts.verbose.

Comment (martinpitt) [reply test/common/testlib.py]: I actually started with that, but testlib.py actually doesn't have an `opts.verbose`, only `trace`. But I think only showing the browser startup/shutdown logs with `-t` seems good enough..

Comment (stefwalter) [reply test/common/testlib.py]: Makes sense.

Comment (martinpitt) [reply test/common/testlib.py]: We do the same in the `testvm.Machine` instantiation, BTW:

    machine = machine_class(verbose=opts.trace, networking=networking, image=image, **kwargs)

**Comment by stefwalter** [comment]
LGTM

**Comment by martinpitt** [comment]
The errors on fedora-27 are due to issue #8417, otherwise the tests passed.

**Pull request merged**